mobile/src/services/auth-api/auth-api.service.ts

```diff
@@ -34,17 +35,29 @@ class AuthApi {
         method: HttpMethod.POST,
         contentType: ContentType.JSON,
         payload: JSON.stringify(payload),
+        hasAuth: false,
       },
     );
   }
 
   signIn(payload: UserSignInRequestDto): Promise<UserSignInResponseDto> {
     return this.#http.load(
-      `${this.#apiPrefix}${ApiPath.AUTH}${AuthApiPath.SIGN_UP}`,
+      `${this.#apiPrefix}${ApiPath.AUTH}${AuthApiPath.SIGN_IN}`,
       {
         method: HttpMethod.POST,
         contentType: ContentType.JSON,
         payload: JSON.stringify(payload),
+        hasAuth: false,
+      },
+    );
+  }
+
+  getCurrentUser(): Promise<UsersByIdResponseDto> {
+    return this.#http.load(
+      `${this.#apiPrefix}${ApiPath.AUTH}${AuthApiPath.CURRENT_USER}`,
+      {
+        method: HttpMethod.GET,
+        contentType: ContentType.JSON,
       },
```

Comment:
get is default parameter and we don't need content type in get requests because we don't send anything
```suggestion
```